pkg/deb/usr/lib/systemd/system/peerbanhelper.service

```diff
@@ -1,11 +1,13 @@
 [Unit]
 Description=PeerBanHelper Service
 After=network.target
+
 [Service]
 User=peerbanhelper
 WorkingDirectory=/etc/peerbanhelper
-ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/etc/peerbanhelper -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
+ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/opt/peerbanhelper -Dpbh.configdir=/etc/peerbanhelper -Dpbh.logdir=/var/log/peerbanhelper -Dpbh.log.level=WARN -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
```

Comment:
@Gaojianli I think this change should be implemented, but will require migration of existing files.

Reply:
@Anuskuss  this change can be implemented

Reply:
I forgot to change the `WorkingDirectory` (because I didn't test it :P). Is this option actually needed though? Now that the data/config dirs are definded I'm not sure PBH actually does something in relative paths. The only other thing I can think of is setting it to `/usr/lib/peerbanhelper` so you can to `-jar PeerBanHelper.jar` (which seems rather useless).
The JAR should probably also go in `/opt` to be honest.

Reply:
> I forgot to change the `WorkingDirectory` (because I didn't test it :P). Is this option actually needed though? Now that the data/config dirs are definded I'm not sure PBH actually does something in relative paths. The only other thing I can think of is setting it to `/usr/lib/peerbanhelper` so you can to `-jar PeerBanHelper.jar` (which seems rather useless). The JAR should probably also go in `/opt` to be honest.

I am not sure where the JAR should be placed... I googled it and someone tell me I should put it in the `/usr/lib/`. But for /opt, I think this directory is more likely used for compiled software. If I am wrong, please feel free to point it.

Reply:
> I forgot to change the `WorkingDirectory` (because I didn't test it :P). Is this option actually needed though? Now that the data/config dirs are definded I'm not sure PBH actually does something in relative paths. The only other thing I can think of is setting it to `/usr/lib/peerbanhelper` so you can to `-jar PeerBanHelper.jar` (which seems rather useless). The JAR should probably also go in `/opt` to be honest.

You are free to set it up and change it, and if it involves a location that PBH doesn't currently support modifying, I can modify the code to accommodate the startup command line.